docs/source/configuration.rst

```diff
@@ -80,6 +80,40 @@ PC15  N/A        N/A     N/A
 ====  =========  ======  ==============================
 
 
+.. index:: interruptions
+
+Exceptions
+==========
+
+The configuration for interruptions (IRQ) and SysTick exceptions is described below. For more information have a look at the `STM32F10xxx/20xxx/21xxx/L1xxxx Cortex®-M3 programming manual`_
+
+==========  =========  =======  ========  =========================
+Exceptions  Excep num  IRQ num  Priority  Functionality
+==========  =========  =======  ========  =========================
+SysTick     15	       -1	1 	  Control and algorithm 
+ADC1_2      N/A        18	1         Low level battery warning
+TIM1_UP	    N/A        25	0	  Infrared state machine
+USART3      N/A        39	1         Bluetooth
+==========  =========  =======  ========  =========================
+  
+The prioriy is configurable, the exception and the IRQ numbers are fixed. A lower priority value indicates higher priority and if the software does not configure any priorities, then all exceptions with a configurable priority have a priority of 0. 
+We didn't find any reference on the documentation but for experience it seems that by default SysTick exception has higher priority than any ISR.
+
+Precedence for system handlers:
+
+1. Priority lowest value
+2. Exception number lowest value
+
+Precedence for interrupt service routines:
+
+1. Group priority lowest value
+2. Subpriority lowest value
+3. IRQ number lowest value
+
+The preemption happens just for higher priority exceptions (nothing to do with exception number or subpriority) and there are 16 priority levels.
```

Comment:
I would try to explain this part better (since "Precedence for system handlers)... :sweat_smile:

Reply:
Re-factored ;)